list-and-titles-of-products.md

```diff
@@ -0,0 +1,37 @@
+The purpose of this PR is to discuss and approve the list and correct titles of all products that we will indicate on the website in the near future.
+
+### Financial area
+
+Ubiquity Dollar 
+
+Ubiquity Card
+
+Ubiquity Payments
+
+### Management area
+
+DevPool
+
+DAO OS 
+
+DAO OS Plugins (each plugin is a product):
+
+- Ubiquity Tasks
+- Ubiquity Rewards (reminds endless rewards)
+- TBD (more plugins will be created by Ubiquity and the community)
+
+### Alternative product names if we consider rebranding 
+
+Ubiquity Dollar 
+- Ubiquity USD (uUSD)
+
+Ubiquity Payments
+- Ubiquity Pay
+- Ubiquity Pays (emphasizes that Ubiquity pays)
```

Comment:
> emphasizes that Ubiquity pays

What do you mean?

Reply:
Most contributors wonder whether they will be paid for tasks they complete. "Ubiquity **pays**" is a play on words based on Apple Pay and "A Lannister always **pays** his debts"